### DISCUSSION TITLE
why no option make Shizuku as Device Owner?

### DISCUSSION THREAD

**Discussion by prasetyodedy** [RikkaApps/Shizuku#447] [Ideas]
why this app startup steps need to be performed again after each reboot?

I find great app can grant app to system level as Device Owner like [Hail](https://github.com/aistra0528/Hail/blob/master/README_EN.md) using command
`adb shell dpm set-device-owner `

is it possible for Shizuku to have that option?

**Comment by RikkaW** [comment]
The permissions (or want can it do) of adb and Device Owner are not the same. In terms of totals, adb is much larger than Device Owner.

Device Owner does not fit the requirement for most of them: https://github.com/ThePBone/awesome-shizuku

**Reply by prasetyodedy** [reply]
but how make it permanently, no need to do startup steps to be performed again after each reboot

**Reply by RikkaW** [reply]
Most of apps using Shizuku will NOT work with only Device Owner 🤔

**Reply by prasetyodedy** [reply]
https://github.com/iamr0s/Dhizuku

now already app for sharing Device Owner called Dhizuku
